maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
+- `c` tags in 1.21+ NeoForge should be kept in sync with Fabric's `c` tags as close as possible. New `c` tags or changes to existing `c` tag should by ran by Fabric to ensure they will agree to make the same changes. `c` tags are to be a standard between loaders with Fabric being our main partner in this effort.
+
+  - Failure to meet this expectation will result in the change being undone so NeoForge matches Fabric again and the incident reviewed to understand how we failed to communicate with Fabric.
+
+## Guidelines
+
+- When making a "big-deal" PR, please post a link to the PR in the NeoForge Discord's maintenance-talk channel, ping the maintainer role, and leave a short sentence on what the PR is about.
+
+  - A "big-deal" PR is one that has a significant impact on many systems, changes the project ecosystem, or requires significant coordination with external projects.
+  
+    - An example of a significant impact on many systems, Transfer PR which involved creating a transaction system and refactoring the item/fluid/energy handlers. Many systems affected and massive amount of changes. This is a big deal.
+
+    - An example of a project ecosystem change is the PR to merge ModLauncher code into FancyModLoader to remove the dependency to ModLauncher. We folded one deliberately-external project into our own project. This is a big deal as it changes what is in scope of the NeoForged organization and changed the relation between projects. If two of our own NeoForged projects were to merge or split up into more, this too would be a big deal as maintainers would definitely like to know as keeping track of what project relates to what is important. *(In addition, when cutting out reliance on an external project that we had close relations with, we should notify the owner of the external project before we cut the ties)*
```

Comment:
This feels like trying to carve out a rule to justify why the FML merge should have been more visible. Fair enough, we could have done a better job. But keep in mind that this change has little impact on users of NeoForge, was already discussed many times over the past months, and even had alpha releases on our maven. The only reason why the change had to happen again slightly differently was due to a technical limitation in our current loader code.